code/game/golf/golf.dm

```diff
@@ -27,11 +27,13 @@
 				"<span class='italics'>You hear a ratchet.</span>")
 				src.anchored = 0
 
-/obj/machinery/golfhole/CanPass(atom/movable/mover, turf/target, height=0)
-	if (contents.len >= 3)
-		visible_message("<span class='notice'>The golf hole is full! Try removing golfballs from the hole.</span>")
-		return ..(mover, target, height)
+/obj/machinery/golfhole/Cross(atom/movable/mover, turf/target)
+	. = ..()
 	if (istype(mover,/obj/item/golfball) && mover.throwing  && anchored)
+		if (contents.len >= 3)
```

Comment:
Except I moved the check here

Reply:
yeah I get the it landed in the hole despite that not even being true